saga/sagaState_prop_test.go

```diff
@@ -0,0 +1,215 @@
+package saga
+
+import (
+	"bytes"
+	"github.com/leanovate/gopter"
+	"github.com/leanovate/gopter/gen"
+	"github.com/leanovate/gopter/prop"
+	"testing"
+)
+
+func Test_ValidSagaId(t *testing.T) {
```

Comment:
Yes agree this is probably better as a unit test, will revert.  This was baby's first property based test in Go, but probably more succinct and always forces the error as a unit test.